src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/Website.cshtml

```diff
@@ -59,14 +59,17 @@
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
                 <form class="js-disable-on-submit" asp-action="EvaluateWebsitePage" asp-controller="RoatpGatewayOrganisationChecks">
-                    <input type="hidden" name="ApplicationId" value="@Model.ApplicationId" />
-                    <input type="hidden" name="PageId" value="@Model.PageId" />
-                    <input type="hidden" name="Ukprn" value="@Model.Ukprn" />
-                    <input type="hidden" name="UkrlpLegalName" value="@Model.UkrlpLegalName" />
-                    <input type="hidden" name="ApplicationSubmittedOn" value="@Model.ApplicationSubmittedOn" />
-                    <input type="hidden" name="SourcesCheckedOn" value="@Model.SourcesCheckedOn" />
-                    <input type="hidden" name="SubmittedWebsite" value="@Model.SubmittedWebsite" />
-                    <input type="hidden" name="UkrlpWebsite" value="@Model.UkrlpWebsite" />
+                    @Html.HiddenFor(x => x.ApplicationId)
```

Comment:
Use of deprecated HTML Helper functions. Use tag helpers

Reply:
done